io/disk/fiotest.py

```diff
@@ -212,7 +214,7 @@ def test(self):
         """
         self.log.info("Test will run on %s", self.dirs)
         fio_job = self.params.get('fio_job', default='fio-simple.job')
-        cmd = '%s/fio %s %s --filename=%s' % (self.sourcedir,
+        cmd = '%s/fio %s --filename=%s%s' % (self.sourcedir,
```

Comment:
@KyleMahlkuch Can you please explain why this particular change is required ?

Reply:
Example from my run on iscsi: 
`/var/tmp/avocado_4rvzt9du/fio/fio /root/avocado-fvt-wrapper/tests/avocado-misc-tests/io/disk/fiotest.py.data/fio-simple.job /dev/mapper/mpatha --filename=fiotest-image`
The "no_fs" tests failed with the above command. filename should point to the mpath so that io is going across the iscsi connection. 

My process for running fio without a filesystem is:
/path/to/fio /path/to/job --filename=/path/to/disk
According to the fio man page: 
fio [options] [jobfile]

So my commit is wrong. It should be: `'%s/fio %s --filename=%s'` because I want filename to be `/dev/mapper/mpatha`
Maybe for the "no_fs" runs there should be a seperate fio command line.

Reply:
Ok, getting your idea. The no_fs option has been working with existing code, but not consistently enough.
I will give this one a try, probably Monday, and then merge this one.

If you have a log on which this works fine, please post one so that it also serves as a comparison point.

Reply:
Feel free to update the commit here, and also take care of indentation in the next 2 lines.